README.md

```diff
@@ -66,19 +66,11 @@ npm run setup
 
 ## Launching
```

Comment:
Yeah, good point. The downside of the api-server is that it requires rclnodejs, which requires ros2 to be available at install time. So we have to make sure rmf is sourced before performing `npm install`.

Reply:
Mmm... I don't know if this is relevant but we will have a problem there if someone just wants to contribute to the front end working with mocks.

Reply:
Yeah, that is the main downside now. Technically romi-dashboard only needs the typing information, so an easy fix is to publish the types separately in another package, however that is a rather big CI challenge.

The main problem is the versioning of romi-dashboard and the api server, since romi-dashboard is a webapp, I would prefer it to always match the master branch of the server, and if we publish the types in its own package, that too should also always be in sync with both the api server and romi-dashboard. I have been thinking of converting romi-dashboard to a monorepo, we would have to update some of our toolings and this will be a drastic change that may break other PRs.

I think I will open an issue to get input if we should move to a monorepo.

Reply:
Actually, romi-dashboard will need the api server for e2e, so regardlessly, it would need to install rclnodejs. The only solution is to dockerized the api server and release it publicly, but downloading a prebuilt server has its own problems in regards to version syncing.

A long term solution is to use a dynamic rcl client that does not require prebuilt typesupport, but there isn't one currently available now and building it would require nontrivial effort. It also will be slower than static rcl clients, it doesn't look like a logical solution.

Of course, we can standardize the server and freeze the api so that we can use a prebuilt server. But I would really like to avoid it as the development of the dashboard and server goes hand in hand and restricting ourself to a stable api looks like a good way to cause many versioning problems down the road.

Reply:
Yeah, I think the monorepo and having vendor a prebuilt version of rclnodejs are both good solutions. 

` may break other PRs.`

About this, we can fix it quickly =). Let's merge this one first and after that, we can introduce that change in case we go with that option.